docs/UNINSTALL.md

```diff
@@ -0,0 +1,29 @@
+# Uninstall
+
+Uninstall is handle by the browser natively.
+
+When the extension is **uninstall** we trigger a redirection to the `UNINSTALL_ORIGIN` URL via the `setUninstallURL` hook:
+```js
+// [...]
+browser.runtime.setUninstallURL(
+  `${UNINSTALL_ORIGIN}${buildQueryString({ extensionId })}`
+)
+// [...]
+```
+Here we can a few metrics to better understand the reason.
```

Comment:
Missing a word …

Here we can {ask for ?} a few metrics to better understand the reason.